contrib/docker/.dockeringnore

```diff
@@ -0,0 +1,2 @@
+.git
+/bin/
```

Comment:
I found that there is an inconsistency between `docker` and `podman` regarding handling `.dockerignore`.
While both process it when it is in the repo's root directory, `docker` also allows `<dockerfile-name>.dockerignore`
https://stackoverflow.com/a/57774684/2749989